src/ui/hexedit.cc

```diff
@@ -52,18 +52,20 @@ void HexEdit::recalculateValues() {
   if (dataBytesCount_ + startOffset_ >= 0x100000000LL) {
     addressBytes_ = 8;
   }
-  addressWidth_ = addressBytes_ * 2 * charWidht_ + verticalAreaSpaceWidth_;
+  addressWidth_ = addressBytes_ * 2 * charWidth_ + verticalAreaSpaceWidth_;
 
   // plus one for only addr
   rowsCount_ = (dataBytesCount_ + bytesPerRow_ - 1) / bytesPerRow_ + 1;
   rowsOnScreen_ =
       (viewport()->height() - horizontalAreaSpaceWidth_ * 2) / charHeight_;
 
-  spaceAfterByte_ = charWidht_ / 2;
-  hexAreaWidth_ = bytesPerRow_ * byteCharsCount_ * charWidht_ +
+  // Spacing needs to be odd for proper border alignment.
+  spaceAfterByte_ = (charWidth_ / 3 | 1);
```

Comment:
But what else could we use? Writing "/ 2 * 2 + 1" just to not use "|" doesn't make sense IMO.
Same with parentheses - '/' has very high priority; you don't add parentheses to "1 / 2 + 3" ;)